lib/src/detectors/hyperv_detector.cc

```diff
@@ -0,0 +1,24 @@
+#include <internal/detectors/hyperv_detector.hpp>
+#include <internal/vm.hpp>
+
+using namespace std;
+
+namespace whereami { namespace detectors {
+
+   result hyperv(const sources::cpuid_base& cpuid_source, sources::smbios_base& dmi_source)
+   {
+       result res {vm::hyperv};
+
+       if (cpuid_source.vendor() == "Microsoft Hv") {
```

Comment:
Oh, missed that - should I try to rule out Virtual PC before validating Hyper-V? Or do we care?